packages/web/src/components/ui/Avatar/Avatar.tsx

```diff
@@ -1,15 +1,36 @@
 import React from 'react';
 import styles from './Avatar.module.scss';
+import stylesTooltip from '../Tooltip/Tooltip.module.scss'; // TODO: is it ok to reference external styles?
```

Comment:
I guess it may work as an ad-hoc solution, but I'd like to avoid it in the future